clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -536,6 +536,10 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
 
   mlir::cir::ConstantOp getConstInt(mlir::Location loc, mlir::Type t,
                                     uint64_t C);
+
+  mlir::cir::ConstantOp getConstFP(mlir::Location loc, mlir::Type t,
+                                   llvm::APFloat fpVal);
```

Comment:
If the API doesn't use anything specific from CIRGen, we should instead add this to `clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h` instead. We have some tech debt of things to move over there (as you probably noticed), but new ones should go there directly!